packages/frontend/src/components/screens/MainScreen/MainScreen.tsx

```diff
@@ -487,7 +487,13 @@ function ChatNavButtons({
   lastUsedApps: T.Message[]
 }) {
   const tx = useTranslationFunction()
-  const onClickThreeDotMenu = useThreeDotMenu(chat)
+  const { openContextMenu } = useChatContextMenu()
+  const onClickThreeDotMenu = useCallback(
+    (event: React.MouseEvent) => {
+      openContextMenu(event, [], chat.id, chat)
```

Comment:
Soo, you can pass an empty array? I think `null` would be more explicit. Plus I think this behavior needs a bit of documentation, i.e. when it's OK to pass `null` / `[]`.